esphome/components/online_image/__init__.py

```diff
@@ -69,11 +78,11 @@ def actions(self):
         cg.add_library("pngle", "1.0.2")
 
 
-# New formats can be added here.
 IMAGE_FORMATS = {
     x.image_type: x
     for x in (
         BMPFormat(),
+        JPEGFormat(),
         PNGFormat(),
     )
 }
```

Comment:
Allowing `format: jpg` as well will avoid some swearing.

```suggestion
}
IMAGE_FORMATS.update({"JPG": IMAGE_FORMATS["JPEG"]})
```

Reply:
Done. I've also added a test for that.